CMSgov/onemac
# OY2-18737, OY2-18748, OY2-18775:

Story: https://qmacbis.atlassian.net/browse/OY2-18737
https://qmacbis.atlassian.net/browse/OY2-18748
https://qmacbis.atlassian.net/browse/OY2-18775
Endpoint: https://d1u81c2uo3xpj8.cloudfront.net

### Details
More cards to add to the triage, connect the dummy page when needed

### Changes
- added the three cards to the Triage Choices library
- added the Routes, but they don't have pages, so "page not found" is the landing page

### Test Plan
1. Login as submitting user
2. Navigate to package dashboard
3. Click "New Submission"
4. Select "Waiver Action"
5. notice the cards for Renewals, Amendments, and Appendix K Amendments are listed.
6. Click each one and verify it goes to the "page not found" page
